common/src/main/java/org/entcore/common/http/filter/AbstractActionFilter.java

```diff
@@ -102,7 +99,9 @@ private void authorizeWorkflowAction(JsonObject session, Binding binding,
 				&& actions != null && actions.size() > 0) {
 			for (Object a: actions) {
 				JsonObject action = (JsonObject) a;
-				if (binding.getServiceMethod().equals(action.getString("name"))) {
+				if (binding.getServiceMethod().equals(action.getString("name")) &&
```

Comment:
Voici l'endroit où on peut simplifier, je pense : 
```suggestion
				if (binding.getServiceMethod().equals(action.getString("right"))) {
```

Pour cela, il suffit que `right` devienne le nouveau champ standard.

Cela implique que, dans le préprocesseur d'annotation générant les fichiers JSON, on lui affecte toujours une valeur, en particulier s'il vaut "chaine vide" (qui est la valeur par défaut). Dans ce cas, il doit être valué avec le nom qualifié de la méthode (ça doit être la valeur du champ `name` en théorie)

C'est pour cela que je préférais nommer le champ `right` plutot que `override`
=> `right` doit devenir la valeur standard